cmd/mount2/mount_test.go

```diff
@@ -3,11 +3,15 @@
 package mount2
 
 import (
+	"runtime"
 	"testing"
 
 	"github.com/rclone/rclone/vfs/vfstest"
 )
 
 func TestMount(t *testing.T) {
+	if runtime.NumCPU() <= 2 {
```

Comment:
These tests are just unreliable rather than locking up - it is only `cmd/mount` which does that. So I'd suggest skipping them with `testy.SkipUnreliable(t)` instead :-)

https://github.com/rclone/rclone/blob/f7f4468cbc5cb349d6c6e4b9f70d69e023a8406a/fstest/testy/testy.go#L9-L10

Reply:
done

Reply:
I can see that we don't `export CI` in our scripts:
```
grep -r CI ./Makefile ./.github/workflows/
```

So you depend on github, right?
https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Is this `CI` variable a standard?
Will gitlab/travis/circleci/jenkings set it to?
I guess you'd better add a link to github CI docs in `testy.go` ....